functions/packages/feed-form/src/impl/feed-form-impl.ts

```diff
@@ -188,3 +176,63 @@ export function buildFeedRow(
     [SheetCol.LogoPermission]: formData.hasLogoPermission,
   };
 }
+
+/**
+ * Sends a Slack webhook message to the configured Slack webhook URL
+ * @param {string} spreadsheetId The ID of the Google Sheet
+ */
+async function sendSlackWebhook(spreadsheetId: string) {
+  const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL;
+  const sheetUrl = `https://docs.google.com/spreadsheets/d/${spreadsheetId}/edit`;
+  if (slackWebhookUrl !== undefined && slackWebhookUrl !== "") {
+    const slackMessage = {
+      blocks: [
+        {
+          type: "header",
+          text: {
+            type: "plain_text",
+            text: "New Feed Added",
+          },
+        },
+        {
+          type: "rich_text",
+          elements: [
+            {
+              type: "rich_text_section",
+              elements: [
+                {
+                  type: "emoji",
+                  name: "inbox_tray",
+                },
+                {
+                  type: "text",
+                  text: "  A new entry was received in the OpenMobilityData source updates Google Sheet",
```

Comment:
[silly question]: Why is the space needed here?

Reply:
In the slack message it's a format thing, so that there is room from the emoji

Reply:
For the future, if David (of all people) had to be explained this, that means most people will not understand the reason for the spaces. You should put a comment explaining this. Not now. In the future.